### ISSUE TITLE
Panic rendering bad SGF

### ISSUE THREAD

**Issue by julianandrews** [julianandrews/sgf-render#45]
I used a ',' instead of a ':'. `sgf-render` panicked:

```
(;GM[1]FF[4]SZ[10,99];B[CC])
```

```
thread 'main' panicked at src/lib/goban.rs:362:20:
internal error: entered unreachable code
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by julianandrews** [comment]
I also get this same panic when SZ exceeds 255 (`u8::MAX`).

**Comment by julianandrews** [comment]
Note that for 99 <= SZ <= 255 instead we get:

```
$ echo '(;GM[1]FF[4]SZ[255])' | sgf-render --format text --no-board-labels
Failed to write output: Range too large for use with labels
```

Which, I think, is fine for now since the rendering assumes labels on the east and west sides of the board are at most two characters. I'll worry about how to handle that more gracefully when I figure out a good solution for #43.

**Comment by julianandrews** [comment]
Fixed in 3940432ca1d70181b7fe311b9bf49df63a45e874
